### PR TITLE
Fix OIDC Examples

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

Staging - https://preview-mongodbmongokart.gatsbyjs.io/pymongo/auth-fix/security/authentication/#mongodb-oidc

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Review by jordan-smith721 — APPROVED** [review]
LGTM

File: source/includes/authentication/azure-imds-connection-string.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
 from pymongo import MongoClient
 
-# define properties, URI, and MongoClient
-properties = {"ENVIRONMENT": "azure", "TOKEN_RESOURCE": "<audience>"}
+# define URI and MongoClient
 uri = ("mongodb://<hostname>:<port>/?"
        "username=<Azure identity client ID>"
        "&authMechanism=MONGODB-OIDC"
-       "&authMechanismProperties=properties")
+       "&authMechanismProperties=environment:azure,token_resource:<audience>")
```
Comment (blink1073) [inline source/includes/authentication/azure-imds-connection-string.py:7]: ENVIRONMENT and TOKEN_RESOURCE should be capitalized by convention in all usages

Comment (mongoKart) [reply source/includes/authentication/azure-imds-connection-string.py]: fixed

**Review by blink1073 — APPROVED** [review]
LGTM!

**Pull request merged**